tests/waku_rln_relay/test_wakunode_rln_relay.nim

```diff
@@ -11,12 +11,35 @@ import
   libp2p/protocols/pubsub/pubsub
 import
   waku/[waku_core, waku_node, waku_rln_relay],
-  ../testlib/wakucore,
-  ../testlib/wakunode,
+  ../testlib/[wakucore, futures, wakunode],
   ./rln/waku_rln_relay_utils
 
 from std/times import epochTime
 
+proc buildWakuRlnConfig_versionAware(
+    credIndex: uint,
+    epochSizeSec: uint64,
+    treeFilename: string,
+    userMessageLimit: int = 1,
+): WakuRlnConfig =
+  let treePath = genTempPath("rln_tree", treeFilename)
+  # Off-chain
+  when defined(rln_v2):
```

Comment:
we only use rln_v2 now, so this compiler directive can be removed

Reply:
Great!